src/assets/i18n/en_US.json

```diff
@@ -428,6 +431,57 @@
   "account.quotes.widget.responded.label": "Responded quotes",
   "account.quotes.widget.submitted.label": "Sent requests",
   "account.quotes.widget.view_all_quotes.link": "View all quotes",
+  "account.recurring_order.delete.confirmation": "Your recurring order has been deleted.",
+  "account.recurring_order.delete_dialog.header": "Delete recurring order \"{{0}}\"",
+  "account.recurring_order.details.active.text": "Active",
+  "account.recurring_order.details.additional_info.heading": "Additional information",
+  "account.recurring_order.details.approved.text": "approved by {{name}} on {{date}}",
+  "account.recurring_order.details.billing_address.heading": "Invoice address",
+  "account.recurring_order.details.breadcrumb": "Recurring order",
+  "account.recurring_order.details.buyer_info.heading": "Buyer",
+  "account.recurring_order.details.cost_center.label": "Cost center",
+  "account.recurring_order.details.creation_date.label": "Creation date",
+  "account.recurring_order.details.expired.text": "Ended",
+  "account.recurring_order.details.inactive-by-system.message": "The last planned order could not be placed and the recurring order has been deactivated.",
+  "account.recurring_order.details.inactive.message": "Your recurring order is currently inactive. <a callback=\"activate\">Activate it now</a> to continue receiving regular deliveries and exclusive discounts.",
+  "account.recurring_order.details.inactive.text": "Inactive",
+  "account.recurring_order.details.info_message": "Please note: Orders are created even if prices or the availability of included items have changed. See the order confirmation for details.",
+  "account.recurring_order.details.last_order_date.label": "Last order",
+  "account.recurring_order.details.last_placed_orders.label": "Last 5 orders",
+  "account.recurring_order.details.links.return_to_orders": "Back to recurring orders",
+  "account.recurring_order.details.next_order_date.label": "Next order",
+  "account.recurring_order.details.order_count.label": "Order count",
+  "account.recurring_order.details.order_number.label": "Recurring order no.",
```

Comment:
Do we need to spell "no." out for accessibility reasons so that screen readers read it aloud correctly?

Reply:
We added an additional localization key with the full term for sr-only to solve this.